pyproject.toml

```diff
@@ -38,6 +30,17 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+# TODO: Use cuda-toolkit package dependencies - e.g. cuda-toolkit[curand,nvvm,nvrtc]=13.*
+cu13 = [
+    "cuda-bindings==13.*",
+    "cuda-core==0.3.2,<0.4.0dev0",
+    "cuda-python==13.*",
```

Comment:
We actually don't need to depend on `cuda-python` anymore, since numba-cuda already depends on the needed components.

Reply:
Does it depend on them by depending on cuda-core?